lib/telegram/bot/types/menu_button_default.rb

```diff
@@ -4,7 +4,7 @@ module Telegram
   module Bot
     module Types
       class MenuButtonDefault < Base
-        attribute :type, Types::String.default('default')
+        attribute :type, Types::String.constrained(eql: 'default')
```

Comment:
I'd keep `.default(...)` too. This way we can init class w/o passing `type:` param (i.e. `Telegram::Bot::Types::MenuButtonDefault.new` vs `Telegram::Bot::Types::MenuButtonDefault.new(type: 'default')`.

Reply:
This is required for DRY to select one of multiple types.

This line: https://github.com/atipugin/telegram-bot-ruby/pull/285/files#diff-1ad9bc58f5795c69b35da4cf0b656b1a3a7b2874db65027d8251d55549f39239R90

The Telegram documentation: https://core.telegram.org/bots/api#menubutton

As you can see, there is no general type, just abstract one, and we have to build one of these three basing on `type` value.

Do you have another approach? I've made custom `proc` with `case` inside previously, but it's ugly, believe me.

So, if there is a possibility to construct `MenuButtonDefault` only with `type: default` or without `type` at all (I guess, `eql constraint` doesn't allow it), tell me please.

Reply:
Yeah, now I see. Well, not a big deal, let's get rid of `.default` 👍

Reply:
I was writing specs for `MenuButton` API and found a weird behavior: `setChatMenuButton` returns `ok: true` but does nothing…

I've wrote to `@BotSupport`.

Meanwhile I've understand that `.constrained(eql: '...').default('...')` will works just as we want.

Reply:
I've done `ChatMember` changes. It works nice.

Kind of unfortunately, I was forced to updated `type_attributes.yml` by Rake task to fix related specs, and it brought a lot of other changes, as `thumb => thumbnail` mentioned previously.